logs/handler.go

```diff
@@ -0,0 +1,144 @@
+package logs
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/duty"
+	"github.com/flanksource/duty/models"
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+	"github.com/flanksource/kommons/ktemplate"
+	"github.com/labstack/echo/v4"
+)
+
+func LogsHandler(c echo.Context) error {
+	req := c.Request()
+
+	var form = make(map[string]any)
+	if err := c.Bind(&form); err != nil {
+		return c.JSON(http.StatusBadRequest, api.HTTPError{
+			Error:   err.Error(),
+			Message: "Invalid request body",
+		})
+	}
+
+	componentID, ok := form["id"].(string)
+	if !ok {
+		return c.JSON(http.StatusBadRequest, api.HTTPError{
+			Error:   "'id' field is required",
+			Message: "component id is required",
+		})
+	}
+
+	component, err := duty.GetComponent(req.Context(), db.Gorm, componentID)
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, api.HTTPError{
+			Error:   err.Error(),
+			Message: "failed to get component by the given ID",
+		})
+	}
+
+	selectorName, _ := form["name"].(string)
```

Comment:
Make the name optional - if none specified use the first log selector.

Reply:
@moshloop it's already like that.